### PR TITLE
Updates HA etcd with kubeadm

### LINKED ISSUE #9222: Issue with k8s.io/docs/tasks/administer-cluster/setup-ha-etcd-with-kubeadm/
**This is a...** 
- [ ] Feature Request
- [x] Bug Report

**Problem:**
This page does not mention the correct systemd unit file to use for the kubelet since it is *different* than the one installed by the packages.

**Proposed Solution:**
Include a section about replacing the systemd unit file.

**Page to Update:**
https://kubernetes.io/docs/tasks/administer-cluster/setup-ha-etcd-with-kubeadm/

### PR DESCRIPTION
**Pull request by chuckha**
Adds a missing instruction on how to configure the kubelet. Reported in slack in the #kubeadm channel.

Fixes #9222

Signed-off-by: Chuck Ha <ha.chuck@gmail.com>

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/setup-ha-etcd-with-kubeadm.md [hunk 1]
```diff
@@ -22,9 +22,10 @@ when using kubeadm to set up a kubernetes cluster.
   document assumes these default ports. However, they are configurable through
   the kubeadm config file.
 * Each host must [have docker, kubelet, and kubeadm installed][toolbox].
-* Some infrastructure to copy files between hosts (e.g., ssh).
+* Some infrastructure to copy files between hosts. For example `ssh` and `scp`
+  can satisfy this requirement.
 
-[toolbox]: /docs/setup/independent/install-kubeadm/
+[toolbox]: /docs/tasks/tools/install-kubeadm/
```
Comment (chuckha) [inline content/en/docs/tasks/administer-cluster/setup-ha-etcd-with-kubeadm.md]: The old link is a redirect and this reflects the real location

File: content/en/docs/tasks/administer-cluster/setup-ha-etcd-with-kubeadm.md [hunk 2]
```diff
@@ -22,9 +22,10 @@ when using kubeadm to set up a kubernetes cluster.
   document assumes these default ports. However, they are configurable through
   the kubeadm config file.
 * Each host must [have docker, kubelet, and kubeadm installed][toolbox].
-* Some infrastructure to copy files between hosts (e.g., ssh).
+* Some infrastructure to copy files between hosts. For example `ssh` and `scp`
```
Comment (chuckha) [inline content/en/docs/tasks/administer-cluster/setup-ha-etcd-with-kubeadm.md]: following documentation style guide

File: content/en/docs/tasks/administer-cluster/setup-ha-etcd-with-kubeadm.md [hunk 3]
```diff
@@ -33,9 +34,37 @@ when using kubeadm to set up a kubernetes cluster.
...
+    instructions will not work.
+
+    ```sh
+    rm -rf /etc/systemd/system/kubelet.service.d
+    cat << EOF > /etc/systemd/system/kubelet.service
+    [Unit]
+    Description=The Standalone Kubelet
+
+    [Service]
+    ExecStart=/usr/bin/kubelet --pod-manifest-path=/etc/kubernetes/manifests --allow-privileged=true
```
Comment (detiber) [inline content/en/docs/tasks/administer-cluster/setup-ha-etcd-with-kubeadm.md]: Why not just override the ExecStart of the drop-in with a higher precedence, say `/etc/systemd/system/kubelet.service.d/20-etcd-service-manager`:

```
[Service]
ExecStart=
ExecStart=/usr/bin/kubelet --pod-manifest-path=/etc/kubernetes/manifests --allow-privileged=true
```

Comment (detiber) [reply content/en/docs/tasks/administer-cluster/setup-ha-etcd-with-kubeadm.md]: I've also had luck with just running the `kubeadm alpha phase certs ca` and `kubeadm alpha phase kubeconfig kubelet` commands with the default systemd unit config, but it does create a lot of noise with errors connecting to the api server, I'm not sure if this method resolves that.

**Comment by Bradamant3** [comment]
This one can definitely wait until after release, please. Among other things, we'll sort out the links because the plan is to move some topics around as a fast follow to the release.

File: content/en/docs/tasks/administer-cluster/setup-ha-etcd-with-kubeadm.md (hunk 3 above)
Comment (chuckha) [reply content/en/docs/tasks/administer-cluster/setup-ha-etcd-with-kubeadm.md]: hmmm

I end up with this when testing that out. Reading more about systemd to see if there is another way to do this

Jun 26 18:52:23 ip-172-31-13-12 systemd[1]: kubelet.service: Service has more than one ExecStart= setting, which is only allowed for Type=oneshot services. Refusing.

Comment (chuckha) [reply content/en/docs/tasks/administer-cluster/setup-ha-etcd-with-kubeadm.md]: oh right, i have to clear it first

```
[Service]
ExecStart=
ExecStart=/usr/bin/kubelet --pod-manifest-path=/etc/kubernetes/manifests --allow-privileged=true
```
haha fixed! This is much better, thank you!

**Pull request merged**